presto-iceberg/src/main/java/io/prestosql/plugin/iceberg/IcebergFileWriterFactory.java

```diff
@@ -133,28 +126,43 @@ private IcebergFileWriter createParquetWriter(
             JobConf jobConf,
             ConnectorSession session)
     {
-        Properties properties = new Properties();
-        properties.setProperty(IOConstants.COLUMNS, columns.stream()
```

Comment:
We can remove the `columns` argument from the method since it's no longer used.

Reply:
Removed, along with a couple of places in IcebergPageSink on the way here.